src/logic/LocalMessageCache.tsx

```diff
@@ -113,7 +130,7 @@ class LocalMessageCache {
     }
 
     static remove(slotName: SlotName, id: string): boolean {
-        const queue = getQueue(slotName);
+        const queue = getQueue(slotName, this.errorHandler ? this.errorHandler : undefined);
```

Comment:
I think this could be simplified to:

```js
getQueue(slotName, this.errorHandler);
```

If `errorHandler` is `undefined`, that's what we'll pass, which is what the ternary ends up doing anyway.